packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -618,25 +618,38 @@ const ToggleMenuDrawer = React.forwardRef(
-                                                    <span className='header__menu-mobile-link-text'>
-                                                        {localize('Dark theme')}
-                                                    </span>
-                                                    <ToggleSwitch
-                                                        id='dt_mobile_drawer_theme_toggler'
-                                                        handleToggle={() => toggleTheme(!is_dark_mode)}
-                                                        is_enabled={is_dark_mode}
+                                            <React.Fragment>
+                                                <MobileDrawer.Item
+                                                    onClick={e => {
+                                                        e.preventDefault();
+                                                        toggleTheme(!is_dark_mode);
+                                                    }}
+                                                >
+                                                    <div className={classNames('header__menu-mobile-link')}>
+                                                        <Icon
+                                                            className='header__menu-mobile-link-icon'
+                                                            icon={'IcTheme'}
+                                                        />
+                                                        <span className='header__menu-mobile-link-text'>
+                                                            {localize('Dark theme')}
+                                                        </span>
+                                                        <ToggleSwitch
+                                                            id='dt_mobile_drawer_theme_toggler'
+                                                            handleToggle={() => toggleTheme(!is_dark_mode)}
+                                                            is_enabled={is_dark_mode}
+                                                        />
+                                                    </div>
+                                                </MobileDrawer.Item>
+                                                <MobileDrawer.Item className='header__menu-mobile-theme'>
+                                                    <MenuLink
+                                                        link_to={getStaticUrl('/help-centre')}
+                                                        icon='IcHelpCentre'
+                                                        text={localize('Help centre')}
+                                                        onClickLink={toggleDrawer}
+                                                        changeCurrentLanguage={changeCurrentLanguage}
```

Comment:
Can you put it in a function like `getLanguageRoutes`. And use it in trader's hub as well.